### PR TITLE
feat: added landing page and refactored stylesheets

### PR DESCRIPTION
**Pull request by srishti-maurya**
This is the landing page, the documentation is not added yet!

Screenshots is attached for reference below.

Any suggestion/feedback would be highly appreciated.

View Live at : https://srishti-maurya.github.io/Plan-it-Component-Library/

screencapture-127-0-0-1-5500-Plan-it-Component-Library-index-html-2022-02-01-02_22_50

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,17 +5,86 @@
...
+          style="
+            width: 704px;
+            height: 100px;
+            border: 0;
+            transform: scale(1);
+            overflow: hidden;
+          "
+          sandbox="allow-scripts allow-same-origin"
+        >
+        </iframe>
```
Comment (tanveertkd) [inline index.html]: You can move the iframe styles on one of the common stylesheets. It will keep the html clean.

**Review by tanveertkd — APPROVED** [review]
Just the one suggestion and I'm guessing you're going to add the button links once you're done with the documentation? Other than that this looks good!

**Comment by gopalbbharadva** [comment]
Looks good.

**Review by gopalbbharadva — COMMENTED** [review]
Looks good.

**Comment by preethasri** [comment]
Hey srishti the logo is awesome. One suggestion is instead of blue  font color ,I think black font  color will add beauty to your website

**Review by mimansha-swarup — APPROVED** [review]
suggestion: you can also use rem or em for height and width value
do correct me If I am wrong

File: index.html (hunk 1 above)
Comment (srishti-maurya) [reply index.html]: Thank you for pointing it out. I'll work on it.

**Review by sakshamAK — APPROVED** [review]

**Pull request merged**